finesse/accuracy_functions.py

```diff
@@ -0,0 +1,115 @@
+import statistics
+import datetime
+import csv
+import os
+from collections import namedtuple
+
+OUTPUT_FOLDER = "./finesse/output"
+AccuracyResult = namedtuple("AccuracyResult", ["position", "total_pages", "score"])
+
+def calculate_accuracy(responses_url: list[str], expected_url: str) -> AccuracyResult:
+    position: int = 0
+    total_pages: int = len(responses_url)
+    score: float = 0.0
+    expected_number = int(expected_url.split('/')[-2])
```

Comment:
regexp instead would make it clearer what you're extracting

Reply:
Done